test/powershell/Language/Parser/RedirectionOperator.Tests.ps1

```diff
@@ -45,42 +45,36 @@ Describe "Redirection operator now supports encoding changes" -Tags "CI" {
         }
     }
 
-    # $availableEncodings = "unknown","string","unicode","bigendianunicode","utf8","utf7", "utf32","ascii","default","oem"
-    $availableEncodings = (Get-Command Out-File).Parameters["Encoding"].Attributes.ValidValues
-
+    $availableEncodings = 
+        @([System.Text.Encoding]::ASCII,
+          [System.Text.Encoding]::BigEndianUnicode,
+          (New-Object System.Text.UTF32Encoding($true,$true)),
```

Comment:
Couple things here, I think..

1. This should be accompanied with a comment indicating why it's different from the below UTF32 encoding, since the code itself doesn't really make it clear unless you're familiar with the constructor.
2. Might just be my own preference, but I'd probably also put it closer to the other UTF32 value in the list.
3. It's probably preferable to use `[System.Text.UTF32Encoding]::new(...)` rather than `New-Object` since the cmdlet is generally a good bit slower.

The syntax here can also be simplified; array subexpressions accept newlines as item separators anyway, and semicolons aren't required in PS for line endings:

```powershell
$availableEncodings = @(
    [System.Text.Encoding]::ASCII
    [System.Text.Encoding]::BigEndianUnicode
    # ..... (etc)
)
```

Reply:
Addressed latest commit.  I left the ordering the same since it lines up with the other, similar enumerations in the source (which are actually alphabetically ordered).